ipc/api/src/checkpoint.rs

```diff
@@ -53,22 +57,27 @@ impl Display for QuorumReachedEvent {
 pub struct BottomUpCheckpointBundle {
     pub checkpoint: BottomUpCheckpoint,
     /// The list of signatures that have signed the checkpoint hash
+    #[serde(deserialize_with = "deserialize_vec_bytes_from_vec_hex")]
+    #[serde(serialize_with = "serialize_vec_bytes_to_hex")]
```

Comment:
Should this say `serialize_vec_bytes_to_vec_hex`, so the two have identical names?